### PR TITLE
[MM-18949] prevent context menu to appear when holding the left click

### PR DESCRIPTION
**Pull request by Willyfrog**
#### Summary

prevent holding the left click to behave like a right click on the desktop

#### Ticket Link

[MM-18949](https://mattermost.atlassian.net/browse/MM-18949)

#### Additional Notes

we are using [react-contextmenu](https://github.com/vkbansal/react-contextmenu) which has a default behaviour of showing context on long presses, its [faq](https://github.com/vkbansal/react-contextmenu/blob/master/docs/faq.md) especifies how to disable that

### CODE REVIEW

File: components/copy_url_context_menu/copy_url_context_menu.jsx [hunk 1]
```diff
@@ -51,9 +51,12 @@ export default class CopyUrlContextMenu extends React.Component {
...
 
         const contextMenuTrigger = (
-            <ContextMenuTrigger id={'copy-url-context-menu' + this.props.menuId}>
+            <ContextMenuTrigger
+                id={'copy-url-context-menu' + this.props.menuId}
+                holdToDisplay={-1}
+            >
                 {this.props.children}
-            </ContextMenuTrigger>
+            </ContextMenuTrigger >
```
Comment (deanwhillier) [inline components/copy_url_context_menu/copy_url_context_menu.jsx]: Looks like a space snuck in here. 😛

**Comment by deanwhillier** [comment]
@Willyfrog, I think you need to update the snapshot for that component so the tests will pass on this PR. 🙂

File: components/copy_url_context_menu/copy_url_context_menu.jsx (hunk 1 above)
Comment (Willyfrog) [reply components/copy_url_context_menu/copy_url_context_menu.jsx]: I was testing a code formatter and I have to ditch it for things like this :( saw the other problems but not this one, thanks for spotting it

**Pull request merged**